opentracing-apache-httpclient/src/main/java/io/opentracing/contrib/apache/http/client/HttpHeadersInjectAdapter.java

```diff
@@ -0,0 +1,30 @@
+package io.opentracing.contrib.apache.http.client;
+
+import java.util.Iterator;
+import java.util.Map;
+
+import org.apache.http.HttpRequest;
+
+import io.opentracing.propagation.TextMap;
+
+/**
+ * @author Pavol Loffay
+ */
+class HttpHeadersInjectAdapter implements TextMap {
```

Comment:
Is this really only supposed to be used by classes on the same package? I just came across a situation where I needed to use the InjectAdapter from the JMS integration into my own code, so, it might be a good idea to make this public.